### PR TITLE
Mark the application as EOL

### PR DESCRIPTION
**Pull request by ghost**
This application is no longer maintained.

### CODE REVIEW

**Comment by ghost** [comment]
> Build [128313 failed](https://buildbot.flathub.org/#/builders/6/builds/128313)

> Failed to download sources: module gnome-online-accounts: transfer closed with 268564 bytes remaining to read

**Comment by ghost** [comment]
@bbhtt com.chmouel.gnomeNextMeetingApplet marked as EOL.

Source: https://github.com/flathub/com.chmouel.gnomeNextMeetingApplet/issues/11#issuecomment-2169139437

Fixes: #11

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "end-of-life": "This application is no longer maintained. Due to a lack of time and no longer using GNOME, I am unable to support the gnomeNextMeetingApplet project anymore. - chmouel."
```
Comment (bbhtt) [inline flathub.json]:
```suggestion
  "end-of-life": "This application is no longer maintained."
```

**Comment by bbhtt** [comment]
I think aarch64 has some download issue, restarting should work.

**Comment by ghost** [comment]
> I think aarch64 has some download issue, restarting should work.

@bbhtt Fixed.

**Comment by bbhtt** [comment]
@chmouel if you intend to work on the app again, please open an issue on https://github.com/flathub/flathub/issues to get this repo back.

**Pull request merged**